MinecraftClient/Translations.cs

```diff
@@ -0,0 +1,247 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace MinecraftClient
+{
+    /// <summary>
+    /// Allows to localize the app in different languages
+    /// </summary>
+    /// <remarks>
+    /// By ORelio (c) 2015-2018 - CDDL 1.0
+    /// </remarks>
+    public static class Translations
+    {
+        private static Dictionary<string, string> translations;
+
+        /// <summary>
+        /// Return a tranlation for the requested text
+        /// </summary>
+        /// <param name="msg_name">text identifier</param>
+        /// <returns>returns translation for this identifier</returns>
+        public static string Get(string msg_name, params object[] args)
+        {
+            if (translations.ContainsKey(msg_name))
+            {
+                if (args.Length > 0)
+                {
+                    return string.Format(translations[msg_name], args);
+                }
+                else return translations[msg_name];
+            }
+                
+
+            return msg_name.ToUpper();
+        }
+
+        /// <summary>
+        /// Initialize translations depending on system language.
+        /// English is the default for all unknown system languages.
+        /// </summary>
+        static Translations()
+        {
+            translations = new Dictionary<string, string>();
+
+            /*
+             * External translation files
+             * These files are loaded from the installation directory as:
+             * Lang/abc.ini, e.g. Lang/eng.ini which is the default language file
+             * Useful for adding new translations of fixing typos without recompiling
+             */
+
+            string systemLanguage = CultureInfo.CurrentCulture.ThreeLetterISOLanguageName;
+            string langDir = AppDomain.CurrentDomain.BaseDirectory + Path.DirectorySeparatorChar + "MCC_Lang" + Path.DirectorySeparatorChar;
```

Comment:
MCC creates a `lang` folder for MC translations in `ChatParser.cs`, maybe we could use `lang/mcc` here?

Reply:
Wouldn't it be confused for people?